rotkehlchen/chain/ethereum/manager.py

```diff
@@ -846,3 +847,44 @@ def get_event_timestamp(self, event: Dict[str, Any]) -> Timestamp:
         block_number = event['blockNumber']
         block_data = self.get_block_by_number(block_number)
         return Timestamp(block_data['timestamp'])
+
+    def _get_blocknumber_by_time_from_subgraph(self, ts: Timestamp) -> int:
+        """Queries Ethereum Blocks Subgraph for closest block with at or before given timestamp"""
+        blocks_subgraph = Graph(
```

Comment:
Hey @nixonbali I think this subgraphs instantiation may probably be best done, in the constructor and saved in `self.blocks_subgraph`, since each time you initialize a `Graph()` a new connection is made.

This way you will also need to use `self` in the method and as such will "fix" the linting error of asking you to make this into a staticmethod :)